contracts/mixnet/src/queries.rs

```diff
@@ -139,6 +141,32 @@ pub(crate) fn query_mixnode_delegations_paged(
     ))
 }
 
+pub(crate) fn query_mixnode_reverse_delegations_paged(
+    deps: Deps,
+    delegation_owner: Addr,
+    start_after: Option<IdentityKey>,
+    limit: Option<u32>,
+) -> StdResult<PagedMixReverseDelegationsResponse> {
+    let limit = limit
+        .unwrap_or(DELEGATION_PAGE_DEFAULT_LIMIT)
+        .min(DELEGATION_PAGE_MAX_LIMIT) as usize;
+    let start = calculate_start_value(start_after);
+
+    let delegations = mix_reverse_delegations_read(deps.storage, &delegation_owner)
+        .range(start.as_deref(), None, Order::Ascending)
+        .take(limit)
+        .map(|res| res.map(|entry| String::from_utf8(entry.0).unwrap()))
```

Comment:
This is present in the current deployed contract as well, and it _should_ be fine, as these values are the ones we read from the storage, where we only write them from sender addresses. If the conversion fails, then we should definitely panic, because that means that the storage got corrupted. But I will add the panic on an `expect`, to have more verbosity.